apps/root/src/services/earnService.ts

```diff
@@ -341,11 +365,13 @@ export class EarnService extends EventsManager<EarnServiceData> {
       strategiesArray.map((userStrategy) => userStrategy.strategy)
     );
 
-    this.userStrategies = savedUserStrategies;
+    if (updateStore) {
```

Comment:
can't we update them here, so we only have 1 fetchUserStrategies method?, if there are positions, we update them, if there are not, we add them